json-schemas/schemas/v4_otc_order_schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "/v4OtcOrderSchema",
+    "properties": {
+        "maker": {
+            "$ref": "/addressSchema"
+        },
+        "taker": {
+            "$ref": "/addressSchema"
+        },
+        "txOrigin": {
+            "$ref": "/addressSchema"
+        },
+        "makerAmount": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "takerAmount": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "makerToken": {
+            "$ref": "/addressSchema"
+        },
+        "takerToken": {
+            "$ref": "/addressSchema"
+        },
+        "expiryAndNonce": {
```

Comment:
Because this is supposed to match the `OtcOrder` schema that exists in the smart contract, I think it should be a 1:1 mapping - breaking this up into 3 fields (it's actually `expiry | nonceBucket | nonce`) would mean this schema can no longer be used to validate legitimate OtcOrder JSON